docs/plugins/dotnet-plugins.md

```diff
@@ -44,4 +44,16 @@ Add authentication and authorization to your .NET service using ASP.NET Core Ide
 
 :::info
 This plugin provides a robust and customizable solution for managing user authentication and authorization in your .NET applications generated by Amplication.
-:::
\ No newline at end of file
+:::
```

Comment:
Let's add here the latest addition as well- "Terraform - AWS Database RDS" -
![image](https://github.com/user-attachments/assets/cb7954ff-533d-4516-aba9-66b6b0175453)